content/webapp/views/components/ImageWithTasl/ImageWithTasl.WorkLink.tsx

```diff
@@ -3,23 +3,25 @@ import styled from 'styled-components';
 import { font } from '@weco/common/utils/classnames';
 import Space from '@weco/common/views/components/styled/Space';
 
-const WorkLinkContainer = styled(Space).attrs({
+export const WorkLinkContainer = styled(Space).attrs({
   $v: { size: 'm', properties: ['margin-top'] },
 })`
   display: inline-flex;
-  align-items: center;
+  align-items: baseline;
 `;
 
-const WorkLink = styled.a`
+export const WorkLink = styled.a`
   margin-left: 0.25rem;
   text-decoration-style: dotted;
   text-underline-offset: 26%;
   text-decoration-thickness: 8%;
 `;
 
-const WorkIcon = styled.img`
-  width: 16px;
-  height: 16px;
+export const WorkIcon = styled.img`
+  width: 1em;
+  height: 1em;
+  position: relative;
+  top: 0.15em;
```

Comment:
Obviously I don't love this magic number, but with `align-self: center` the icon would potentially be centred with two (or more) lines of link text instead of centred with the first which is what we want – I think this is the only reasonable way to achieve that

__what we don't want (`align-self: center`)__
<img width="907" height="137" alt="Image" src="https://github.com/user-attachments/assets/2119b3ca-a69e-442f-a15a-cebadf088f50" />